ci/run-wasi-nn-example.sh

```diff
@@ -50,6 +74,15 @@ popd
 cargo run -- run --dir $TMP_DIR::fixture -S nn,nn-graph=openvino::$TMP_DIR \
     $TMP_DIR/wasi-nn-example-named.wasm
 
+# Build and run an ONNX component example doing classification.
+cargo build -p wasi-preview1-component-adapter --target wasm32-unknown-unknown --release
+pushd $WASMTIME_DIR/crates/wasi-nn/examples/classification-component-onnx
+cargo component build --release
+cp target/wasm32-wasi/release/classification_component_onnx.wasm $TMP_DIR
+cp -rf ./fixture $TMP_DIR/
+popd
+cargo run --features wasi-nn,component-model -- run --wasm component-model --dir $TMP_DIR/fixture::fixture -S nn $TMP_DIR/classification_component_onnx.wasm
```

Comment:
I don't think we'll need this script anymore now that #7679 is merged.

Reply:
I fixed this and added a test program.